### PR TITLE
Changed theme and leftover icons and strings for android

### PR DESCRIPTION
**Pull request by Nikhil7174**
WhatsApp Image 2024-05-18 at 2 20 14 AM
WhatsApp Image 2024-05-16 at 2 21 45 AM
Screenshot_20240518-135834_Ping - Debug

### CODE REVIEW

File: build/commands/lib/config.js [hunk 1]
```diff
@@ -523,7 +523,7 @@ Config.prototype.buildArgs = function () {
     args.android_channel = this.channel
     if (!this.isReleaseBuild()) {
```
Comment (udaybansal19) [inline build/commands/lib/config.js:524]: This change will only reflect in non release builds

btw why is this needed and what might be impacted by changing this?

Comment (Nikhil7174) [reply build/commands/lib/config.js:524]: Changed package name for all builds. 
It is needed to change the package name which is also visible in ui (dowloads settings), screenshot provided above.

**Review by udaybansal19 — APPROVED** [review]
